frontend/src/semantic-ui/site/elements/button.variables

```diff
@@ -1,6 +1,12 @@
 /*******************************
     User Variable Overrides
 *******************************/
-@primaryTextColor   : @white;
+
+@background         : white;
```

Comment:
Will this affect other buttons that have non-white background color?

Reply:
yes it will. I have updated the code and deleted this line